wl-clipboard-tools/Cargo.toml

```diff
@@ -0,0 +1,60 @@
+[package]
+name = "wl-clipboard-tools"
+version = "0.4.1" # remember to update html_root_url
+authors = ["Ivan Molodetskikh <yalterz@gmail.com>"]
+description = "Access to the Wayland clipboard for terminal and other window-less applications."
+edition = "2018"
+license = "MIT/Apache-2.0"
+
+readme = "README.md"
+documentation = "https://docs.rs/wl-clipboard-rs"
+repository = "https://github.com/YaLTeR/wl-clipboard-rs"
+keywords = ["wayland", "clipboard"]
+categories = ["command-line-utilities"]
+
+[badges]
+travis-ci = { repository = "YaLTeR/wl-clipboard-rs" }
+
+# [profile.dev]
+# panic = "abort"
+
+[dependencies]
+wl-clipboard-rs = { path = "../" }
+derive-new = "0.5"
```

Comment:
I think some of this stuff is needed only for the library and not for the binaries?

Reply:
fixed